intellij-plugin/src/main/kotlin/com/apollographql/ijplugin/inspection/GraphQLUnresolvedReferenceInspectionSuppressor.kt

```diff
@@ -26,8 +27,9 @@ class GraphQLUnresolvedReferenceInspectionSuppressor : InspectionSuppressor {
 
       "GraphQLMissingType" -> element is GraphQLDirectivesAware && element.directives.all { it.isKnownDirective() }
 
-      // We need to suppress this one too because the plugin doesn't know that @link is repeatable
-      "GraphQLDuplicateDirective" -> element is GraphQLDirective && element.name == "link"
+      // We need to suppress this one too because the plugin doesn't know that certain directives (e.g. @link) are repeatable
+      "GraphQLDuplicateDirective" -> element is GraphQLDirective &&
+          (element.name == "link" || KNOWN_DIRECTIVES.any { it.name == element.name && it.repeatable })
```

Comment:
Can this be simplified to:

```suggestion
          KNOWN_DIRECTIVES.any { it.name == element.name && it.repeatable }
```
?
Looks like `@link` is part of KNOWN_DIRECTIVES